### PR TITLE
more friendly bot messages

### PR DESCRIPTION
**Pull request by kieferrm**
Our bot messages are rather formal. Here is a proposal for less formal, more friendly ones.

### CODE REVIEW

File: .github/commands.yml [hunk 1]
```diff
@@ -5,25 +5,25 @@
...
         },
         {
             type: 'label',
             name: '*extension-candidate',
             action: 'close',
-            comment: "This issue has been closed because it is not within the scope of the core product, but could be addressed by an extension. The [VS Code Marketplace](https://aka.ms/vscodemarketplace) shows all existing extensions and you can get started [writing your own extension](https://aka.ms/vscodewritingextensions) in a few simple steps. See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.\n\nHappy Coding!"
+            comment: "We try to keep VS Code lean and we think the functionality you're asking for is great for a VS Code extension. Maybe you can already find one that suits your in the [VS Code Marketplace](https://aka.ms/vscodemarketplace). Just in case, in a few simple steps you can get started [writing your own extension](https://aka.ms/vscodewritingextensions). See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.\n\nHappy Coding!"
```
Comment (chrmarti) [inline .github/commands.yml]: "suits you" or "suits your needs"?

File: .github/commands.yml [hunk 2]
```diff
@@ -5,25 +5,25 @@
...
         },
         {
             type: 'label',
             name: '*not-reproducible',
             action: 'close',
-            comment: "This issue has been closed because the problem could not be reproduced either because it is already fixed in later versions of the product or because it requires additional details on how to reproduce it. See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.\n\nHappy Coding!"
+            comment: "We closed this issue because we are unable to reproduce the problem with the steps you describe. Chances are we've already fixed your problem in a recent version of VS Code. If not, please reopen the issue and provide us with more detail. Our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines might help you with that.\n\nHappy Coding!"
```
Comment (chrmarti) [inline .github/commands.yml]: "ask us to reopen"? I think only we can do that.

File: .github/commands.yml (hunk 1 above)
Comment (kieferrm) [reply .github/commands.yml]: Either works for me.

File: .github/commands.yml (hunk 2 above)
Comment (kieferrm) [reply .github/commands.yml]: Yup. "Ask us to reopen".

**Pull request merged**